styles/styles.css

```diff
@@ -58,6 +60,7 @@ body {
   margin: 0;
   font-family: var(--body-font-family);
   line-height: 1.6;
+  text-align: center;
```

Comment:
Making the entire site's body text centered might not be desirable, but we can leave it.